src/metabase/sync_database/analyze.clj

```diff
@@ -251,6 +169,7 @@
         finished-tables-count (atom 0)]
     (doseq [{table-name :name, :as table} tables]
       (try
+        #_(cached-values/cache-table-data-shape! driver table)
```

Comment:
not sure. The expected logic is 
1. field values are cached on a schedule
2. When first setting up a DB we should do all three actions (sync, cache-values, analyze->classify)
3. Any time something is analyzed it should be fed to the classifier

In the future we'll tweak all of these.